python/Interscale_hub/pivot.py

```diff
@@ -40,8 +40,13 @@ def __init__(self, param, comm_receiver, comm_sender, databuffer):
         '''
         
         # TODO: logger placeholder for testing
+        import sys
         self.__logger = logging.getLogger(__name__)
-        self.__logger.info("Initialise...")
+        handler = logging.StreamHandler(sys.stdout)
+        formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
```

Comment:
Isn't this something that is configurable? It feels somewhat repeated to have the same rather complex line on multiple locations